### PR TITLE
[Fix #58] Enable RSpec/LetSetup by default

### LINKED ISSUE #58: Rspec: RSpec/LetSetup - not to use `let!`
[This cop](https://rubocop-rspec.readthedocs.io/en/latest/cops_rspec/#rspecletsetup) does not allow to use`let!` for setting up test data. 

By documentation, it should be disabled by default but I caught this issue in the project.

```
spec/services/review_status/move_to_published_spec.rb:14:5: C: RSpec/LetSetup: Do not use let! for test setup.
    let!(:status) { create :review_status, workflow: workflow, status: :approved, created_by: user }
    ^^^^^^^^^^^^^
spec/services/review_status/move_to_published_spec.rb:34:7: C: RSpec/LetSetup: Do not use let! for test setup.
      let!(:status) { create :review_status, workflow: workflow, status: :pending, created_by: user }
      ^^^^^^^^^^^^^
```

Any thoughts? I didn't investigate why it was added to the default config so I don't have any thoughts now.

### PR DESCRIPTION
**Pull request by roman-dubrovsky**
Before you submit a pull request, please make sure you have to follow:

- [x] read and know items from the [Contributing Guide](https://github.com/datarockets/datarockets-style/blob/master/CONTRIBUTING.md#pull-requests)
- [x] add a description of the problem you're trying to solve (short summary from related issue)
- [x] verified that cops are ordered by alphabet
- [ ] add a note to the style guide docs (if it needs)
- [x] add a note to the changelog file
- [x] the commit message contains the number of the related issue (if it presents)
  and word `Fix` if this PR closes related issue
- [x] squash all commits before submitting to review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,11 @@ There are some areas in which there is no clear consensus in datarockets team re
 In such scenarios, all popular styles are acknowledged and it’s up to you to pick one and apply it consistently.
 For that just set up these cops before starting of usage.
 
+#### RSpec/LetSetup
+
+[This cop](https://rubocop-rspec.readthedocs.io/en/latest/cops_rspec/#rspecletsetup) is enabled by default and we suggest not to use unreferenced `let` variables in your test cases.
+But you feel that unreferenced `let` variables are needed and test can't be written w/p them, please create an issue with your case so that we can re-thinking our solution about enabling this cop.
```
Comment (atatarenko) [inline README.md]: > But you feel that unreferenced `let` variables are needed

To

> However, if it feels like the cop should be enabled

?

Comment (atatarenko) [reply README.md]: `be written w/p them` -> `be written w/o them`

Comment (atatarenko) [reply README.md]: `and test can't be` -> `and a test can't be`

Comment (roman-dubrovsky) [reply README.md]: 👍

Comment (roman-dubrovsky) [reply README.md]: P.S. `and test can't be` -> `and tests can't be`

**Pull request merged**